webservices/rest.py

```diff
@@ -165,6 +168,16 @@ def limit_remote_addr():
                 request_api_key_id = request.headers.get('X-Api-User-Id')
                 if request_api_key_id != DOWNLOAD_WHITELIST_API_KEY_ID:
                     abort(403)
+            # We don't want to accidentally block downloads here, handled above
+            elif RESTRICT_API_TRAFFIC in true_values and '/v1/' in request.url:
+                request_api_key_id = request.headers.get('X-Api-User-Id')
+                if request_api_key_id not in TRAFFIC_WHITELIST_API_KEY_IDS:
+                    abort(
+                        503,  # Service unavailable
+                        "We apologize for the inconvenience, but we are temporarily "
```

Comment:
Can you put this message in a constant variable and use that variable here?

Reply:
@pkfec thanks, I think that improves readability. Implemented with d542538c593d3eadc0dde2279dabf4d43771af15 and deploying now.

Reply:
looks good.